### ISSUE TITLE
ios compilation errors after upgrading Flutter to 2.2

### ISSUE THREAD

**Issue by alex9153** [flutter/flutter#83353]
```
Error output from Xcode build:
↳
    ** BUILD FAILED **
Xcode's output:
↳
    FirebaseAnalytics-euifhorukdnxydevwsudntldhlte
    Command PhaseScriptExecution failed with a nonzero exit code
    FirebaseAnalytics-euifhorukdnxydevwsudntldhlte
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Analyzing workspace
    note: Constructing build description
    note: Build preparation complete
    /Users/aleks/Projects/WORK/voice_new_design/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.5.99. (in target 'AppAuth' from project 'Pods')
    /Users/aleks/Projects/WORK/voice_new_design/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.5.99. (in target 'FBSDKCoreKit' from project 'Pods')
    /Users/aleks/Projects/WORK/voice_new_design/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.5.99. (in target 'FBSDKLoginKit' from project 'Pods')
    /Users/aleks/Projects/WORK/voice_new_design/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.5.99. (in target 'FMDB' from project 'Pods')
```

**Comment by hnvn** [comment]
Have a look at [this](https://github.com/flutter/flutter/issues/80978)

**Comment by TahaTesser** [comment]
Hi @alex9153 
Closing as duplicate of https://github.com/flutter/flutter/issues/80978
